### ISSUE TITLE
Use platforms' native keychains for caching tokens rather than config.json

### ISSUE THREAD

**Issue by armarti** [Keeper-Security/Commander#292]
All new-ish versions of macOS / Windows / Linux have some convenient and free way of securely storing credentials, either included with the OS or easily installable. It'd be neat to use those instead of caching tokens in `config.json`.

**Comment by armarti** [comment]
Or perhaps you could just build one solid integration with `pass` since it works for Windows [too](https://www.passwordstore.org/#other).

**Comment by b1tninja** [comment]
Definitely. I was looking at [keyring](https://pypi.org/project/keyring/) it would support Windows Credential Manager, MacOS Keychain, and Freedesktop SecretService

**Comment by craiglurey** [comment]
We created a keyring utility: https://github.com/Keeper-Security/linux-keyring-utility
It isn't directly integrated into Commander, but someone could decide to do it.
